visualization/app/codeCharta/ui/fileChooser/fileChooser.component.spec.ts

```diff
@@ -46,7 +46,7 @@ describe("fileChooserController", () => {
 			loadFiles: jest.fn().mockReturnValue({ catch: jest.fn() })
 		})()
 	}
-
+	// TODO add mocking setNewData with sortingArray and remove the upper test
```

Comment:
Has this been addressed, or can we remove the comment?

Reply:
yes we can :)

Reply:
```suggestion
```